spec/features/operator/clusterConfig/clusterConfig_test.go

```diff
@@ -0,0 +1,29 @@
+package clusterConfig
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestClusterConfigReinstall(t *testing.T) {
+	Convey("Given the Operator is installed", t, func() {
+		Convey("Delete cluster config from cluster", nil)
+		Convey("Verify pipeline, Triggers, ClusterTasks has been uninstalled or not", nil)
+		Convey("Create new config name 'Cluster' with target namespace 'openshift-pipelines'", nil)
```

Comment:
`It ` below should move inside this block

Reply:
Done